app/assets/stylesheets/components/_option-select.scss

```diff
@@ -107,10 +107,17 @@
 
 .app-c-option-select__filter {
   display: none;
+  padding-top: govuk-spacing(2);
+  margin-bottom: govuk-spacing(2);
+}
+
+.app-c-option-select__filter-input {
+  padding-left: 33px;
+  background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 36 36' width='40' height='40'%3E%3Cpath d='M25.7 24.8L21.9 21c.7-1 1.1-2.2 1.1-3.5 0-3.6-2.9-6.5-6.5-6.5S10 13.9 10 17.5s2.9 6.5 6.5 6.5c1.6 0 3-.6 4.1-1.5l3.7 3.7 1.4-1.4zM12 17.5c0-2.5 2-4.5 4.5-4.5s4.5 2 4.5 4.5-2 4.5-4.5 4.5-4.5-2-4.5-4.5z' fill='currentColor'%3E%3C/path%3E%3C/svg%3E") govuk-colour("white") no-repeat -5px -3px;
+  @include govuk-font(19);
 
-  .govuk-form-group {
-    padding-top: govuk-spacing(2);
-    margin-bottom: govuk-spacing(2);
+  @include media(tablet) {
```

Comment:
I'm aware we're trying to move to GOV.UK Frontend. Can we use the GOV.UK Frontend media query mixin instead? https://govuk-design-system-preview.netlify.com/get-started/updating-your-code/